### PR TITLE
Created serializers for User and Coder tables

### PR DESCRIPTION
**Pull request by dmytrosmetanin**
Pls check my changes, comment if anything and approve if everything's fine.

### CODE REVIEW

File: codeArena/user/serializers.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from rest_framework import serializers
+from user.models import User, Coder
+
+
+class UserSerializer(serializers.ModelSerializer):
```
Comment (MrSancho42) [inline codeArena/user/serializers.py:8]: Vitaliy recommended use serializer.Serializer for complex tables, couse it more flexible

File: codeArena/user/serializers.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+def update(self, instance, validated_data):
+    if "password" in validated_data:
+        from django.contrib.auth.hashers import make_password
+        validated_data["password"] = make_password(validated_data["password"])
+    return super().update(instance, validated_data)
+
+
+class CoderSerializer(serializers.ModelSerializer):
```
Comment (MrSancho42) [inline codeArena/user/serializers.py:33]: In this case same issue as higher

File: codeArena/user/serializers.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    )
+
+    class Meta:
+        model = User
+        fields = ['id', 'email', 'nickname', 'first_name', 'last_name',
+                  'password', 'role_id', 'created_at', 'updated_at', 'status']
+
+
+def create(self, validated_data):
+    if "password" in validated_data:
```
Comment (MrSancho42) [inline codeArena/user/serializers.py]: PEP8 recommend use single quotes

**Review by MrSancho42 — CHANGES_REQUESTED** [review]
Are .create(), and .update() UserSerializer's methods?

File: codeArena/user/serializers.py (hunk 1 above)
Comment (vvelizariy) [reply codeArena/user/serializers.py:8]: For user it can be used. As we will not manipulate on user fields.

File: codeArena/user/serializers.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+def create(self, validated_data):
+    if 'password' in validated_data:
+        from django.contrib.auth.hashers import make_password
+        validated_data['password'] = make_password(validated_data['password'])
+    return super(UserSerializer, self).create(validated_data)
+
+
+def update(self, instance, validated_data):
+    if 'password' in validated_data:
+        from django.contrib.auth.hashers import make_password
```
Comment (vvelizariy) [inline codeArena/user/serializers.py]: As this import is used more than once, move it to global imports on top

**Review by MetalMari — APPROVED** [review]

**Review by s0fko — APPROVED** [review]

**Review by Viktorika-p — APPROVED** [review]

**Review by MrSancho42 — APPROVED** [review]

**Pull request merged**